src/internal/td.rs

```diff
@@ -129,9 +139,9 @@ impl<'hdr> TagData<'hdr> {
     }
 
     /// Obtain a char value, if this is a char
-    pub fn as_char(&self) -> Option<&[char]> {
+    pub fn as_char(&self) -> Option<char> {
```

Comment:
Since you are no longer borrowing here, but making a conversion to an owned type, the Rust idiom would be `to_*`:

```suggestion
    pub fn to_char(&self) -> Option<char> {
```

...and this goes for all of the rest of the `as_*` methods as well.

Reply:
indeed ...

Reply:
done